Testing/VelaptorTests/VelaptorTests.csproj

```diff
@@ -1,119 +1,108 @@
+		</None>
+		<None Update="SampleTestData\BatchItemTestData\RenderFont_WhenInvoking6ParamsWithColorOverload_RendersFont.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+		<None Update="SampleTestData\BatchItemTestData\RenderFont_WhenInvoking5ParamsWithPositionOverload_RendersFont.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+		<None Update="SampleTestData\BatchItemTestData\RenderFont_WhenInvoking5ParamsWithColorOverload_RendersFont.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+		<None Update="SampleTestData\BatchItemTestData\RenderFont_WhenInvoking5ParamsWithPositionAndColorOverload_RendersFont.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+		<None Update="SampleTestData\RectGPUDataTestData.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+		<None Update="SampleTestData\GenerateData_WhenInvoked_ReturnsCorrectResult_TestData.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+		<None Update="SampleTestData\GenerateIndices_WhenInvoked_ReturnsCorrectResult_TestData.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+	</ItemGroup>
+	<ItemGroup>
+		<Compile Remove="Fakes\AudioManagerFake.cs"/>
+	</ItemGroup>
+	<ItemGroup>
+		<None Remove="stylecop.json"/>
+	</ItemGroup>
+	<ItemGroup>
+		<AdditionalFiles Include="stylecop.json"/>
+	</ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="coverlet.msbuild" Version="3.1.2">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+		<PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0"/>
+		<PackageReference Include="Moq" Version="4.18.2"/>
+		<PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+		<PackageReference Include="System.IO.Abstractions" Version="13.2.43"/>
+		<PackageReference Include="xunit" Version="2.4.1"/>
+		<PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+	</ItemGroup>
+	<ItemGroup>
+		<Folder Include="TestAssets\"/>
+	</ItemGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\..\Velaptor\Velaptor.csproj"/>
+	</ItemGroup>
+	<ItemGroup>
+		<None Update="TestAssets\TestCompareImage.png">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</None>
+	</ItemGroup>
+</Project>
```

Comment:
Please add a single empty line at the bottom of the file.

Every file should have this.  It is not the end of the world, but the GitHub ecosystem recommends this, so I follow it.

This has to do with some historic reasoning with UNIX type operating systems.

Reply:
```suggestion
</Project>

```

Reply:
DOne :+1: